doc/ref_model/chapters/chapter03.md

```diff
@@ -539,6 +542,150 @@ SyncE was standardized by the ITU-T, in cooperation with IEEE, as three recommen
 * ITU-T Rec. G.8264 that describes the specification of Ethernet Synchronization Messaging Channel (ESMC)
 SyncE architecture minimally requires replacement of the internal clock of the Ethernet card by a phase locked loop in order to feed the Ethernet PHY. 
 
+
+<a name="3.5.7"></a>
+ ### 3.5.7 Kubernetes Networking Semantics
+The support for traditional network orchestration is more or less non existing in Kubernetes proper. Kubernetes is foremost a Platform as a Service (PaaS) environment and not an Infrastrutcure as a Service (Iaas) infrastucture component. There is no orchetration API like Neutron and no way to instantiate network services such as L3aaS and LBaaS like you can do in Openstack.
+
+Kubernetes networking can be divided into two parts, built in network functinality available through the pod's mandatory primarty interface and network functionality avaiable through the pod's optional secondary inerfaces.
+
```

Comment:
In ideal cases this division would not exist and Kubernetes would handle all interfaces equaly. At the moment the secondary interfaces are not supported by Kubernetes at all.

Reply:
I rewrote part of this, and as for the multi networking, I hope that this will be non issue within 2-3 years